client/src/components/MapController/MapController.js

```diff
@@ -30,6 +30,13 @@ var MapController = React.createClass({
     var update = {}; update[param] = selection;
     this.setState(update);
   },
+
+  handleSetArea: function(wkt) {
+    // TODO: Do something more here?
+    // Really just here so it's not invisibly transferred from parent props, not directly needed
```

Comment:
@jameshiebert Thoughts? This was added last minute because I didn't like to invisibly pass an essential property from ancestor to child through `{...this.props}`